assets/src/domain/eventEditor/ui/datetimes/datesList/tableView/useBodyRowGenerator.tsx

```diff
@@ -34,12 +34,11 @@ const useBodyRowGenerator = (): DatesTableBodyRowGen => {
 		const name = {
 			key: 'name',
 			type: 'cell',
-			className:
-				'ee-date-list-cell ee-date-list-col-name ee-rspnsv-table-column-bigger ee-rspnsv-table-hide-on-mobile',
+			className: 'ee-date-list-cell ee-col-name ee-rspnsv-table-column-bigger ee-rspnsv-table-hide-on-mobile',
```

Comment:
`ee-date-list-col-name` wasn't very good because I'd have to target 2 different selectors(one for date and one for ticket), that's why I've replaced it with `ee-col-name`

Reply:
great 💯